master/reference/kube-controllers/configuration.md

```diff
@@ -80,6 +81,12 @@ To enable the node controller, perform the following two steps.
 
 This controller is only valid when using etcd as the {{site.prodname}} datastore.
 
+Additionally, the node controller can be configured to keep labels on Kubernetes node
+objects and Calico node objects in sync. By setting `SYNC_NODE_LABELS` to true (enabled
```

Comment:
Here we are still seeing "Calico node" on lines 85, 87, and 88.  I thought perhaps we would go to 
```
`{{site.nodecontainer`
```?